.github/workflows/ci-ops-infra.yml

```diff
@@ -0,0 +1,68 @@
+name: "CI - Update OPS Infrastructure"
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/ci-ops-infra.yml'
+      - '.github/actions/deploy-terraservice/**'
+      - 'ops/terraform/env/mgmt/**'
+
+  workflow_dispatch:
+permissions:
+  id-token: write # This is required for requesting the AWS IAM OIDC JWT
+  contents: write # This is required for actions/checkout
+defaults:
+  run:
+    shell: bash
+env:
+  DEFAULT_LOG_GROUP: "/bfd/mgmt/gha/ci-ops-infra"
+  DEFAULT_LOG_STREAM: "deploy-terraservice_${{ github.run_number }}"
+
+jobs:
+  update-mgmt-infra:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref_name }}
+
+      - name: Configure AWS credentials
+        id: config-aws-creds
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.GHA_AWS_IAM_ROLE_ARN }}
+          role-session-name: ci-deploy-mgmt
+          aws-region: us-east-1
+
+      - name: Mask sensitive AWS data
+        id: mask-sensitive-aws-data
+        run: |
+          caller_id="$(aws sts get-caller-identity)"
+          account_num="$(jq -r '.Account' <<<${caller_id})"
+          role_arn="$(jq -r '.Arn' <<<${caller_id})"
+          user_id="$(jq -r '.UserId' <<<${caller_id})"
+
+          echo "::add-mask::$account_num"
```

Comment:
Do we want to add `add-mask` for caller_id too?

Reply:
> Do we want to add `add-mask` for caller_id too?

I don't think that's necessary. I wouldn't expect `caller_id` to appear in the logs, and logs including `caller_id` in stdout won't leak any sensitive information due to the existing masking of its components. If it were to appear, it would as the following:
```
{
  "UserId": ***,
  "Account": ***,
  "Arn": ***
}
```